cmd/dendrite-edu-server/main.go

```diff
@@ -29,8 +29,9 @@ func main() {
 			logrus.WithError(err).Warn("BaseDendrite close failed")
 		}
 	}()
+	deviceDB := base.CreateDeviceDB()
```

Comment:
Doesn't this mean more than one component is now touching the device DB?

Reply:
Just about all of the components touch the device DB in some way or other already - it really needs the same kind of treatment as the server key API got.